neat-screen.js

```diff
@@ -5,10 +5,9 @@ var strftime = require('strftime')
 var views = require('./views')
 var util = require('./util')
 var markdown = require('./markdown-shim')
-var welcomeMessage = ['welcome to cabal',
-  'type /channels to see which channels to join, and /help for more commands',
-  'for more info visit https://github.com/cabal-club/cabal']
-
+var fs = require("fs")
+var welcomeMessage = fs.readFileSync("welcome.txt").toString().split("\n")
```

Comment:
```suggestion

var welcomePath = path.join(__dirname, 'welcome.txt')
var welcomeMessage = fs.readFileSync(welcomePath, 'utf8').split("\n")
```

Reply:
`__dirname` captures whereever `cli.js` and the rest of the code and resources are. Otherwise, if you ran `cabal` from `~/foo` it would look for `~/foo/welcome.txt`.

Reply:
yay code review catches the bugz, thanks friend :3 

pushing an update in a moment